src/test/groovy/edu/ie3/datamodel/models/StandardLoadProfileTest.groovy

```diff
@@ -0,0 +1,84 @@
+package edu.ie3.datamodel.models
+
+import spock.lang.Specification
+
+class StandardLoadProfileTest extends Specification {
+    def "Standard load profiles are parsed correctly from correct input" () {
+        when:
+        StandardLoadProfile actual = StandardLoadProfile.parse(key)
+
+        then:
+        actual == expected
+
+        where:
+        key     || expected
+        "h0"    || BdewLoadProfile.H0
```

Comment:
👍  for this test - documents all allowed cases for other developers/users and also tests them - nice work!